components/ai_chat/content/browser/page_content_fetcher.cc

```diff
@@ -244,6 +324,16 @@ void FetchPageContent(content::WebContents* web_contents,
     return;
   }
 
+#if BUILDFLAG(ENABLE_TEXT_RECOGNITION)
+  auto host = web_contents->GetURL().host();
+  if (base::Contains(kScreenshotRetrievalHosts, host)) {
+    if (FetchPageContentViaTextExtraction(web_contents, std::move(callback))) {
+      // Callback already handled
+      return;
```

Comment:
Good catch, this function doesn't actually need to exist so I'll just remove it so it can't happen. It would only return false if the view couldn't be created before which I don't know if would ever actually happen. But it was definitely a bug.